File: Java/Perro.java
/*
                  Universidad de las fuerzas armadas
                   Tecnologías de la información
Nombre: Juan Pablo Cano                             NRC: 13930
Fecha: 13-11-2023
                            
Autor: Juan Pablo Cano 
 */
package gato_perro;

public class Perro {   
    String nombre;
    int edad;
    String raza;

    public Perro(String nombre, int edad, String raza) {
        this.nombre = nombre;
        this.edad = edad;
        this.raza = raza;
    }
    
    public void comer (){
        System.out.println(this.nombre+" empezo a comer"); 
        
    }
    
        public void saltar (){
        System.out.println(this.nombre+" empezo a comer"); 
        
    }
        public void ladrar (){
        System.out.println(this.nombre+" empezo a comer"); 
        
    }
    
    
    
    
}

File: Primer Parcial/C++/Perro.cpp
#include <iostream>
#include <string>

class Perro {
private:
    std::string nombre;
    int edad;
    std::string raza;

public:
    Perro(std::string nombre, int edad, std::string raza) : nombre(nombre), edad(edad), raza(raza) {}

    void comer() {
        std::cout << nombre << " empezo a comer" << std::endl;
    }

    void saltar() {
        std::cout << nombre << " empezo a saltar" << std::endl;
    }

    void ladrar() {
        std::cout << nombre << " empezo a ladrar" << std::endl;
    }
};

int main() {
    // Ejemplo de uso
    Perro miPerro("Buddy", 5, "Golden Retriever");
    miPerro.comer();
    miPerro.saltar();
    miPerro.ladrar();

    return 0;
}
